### ISSUE TITLE
Versioning for omSupply central server

### ISSUE THREAD

**Issue by andreievg** [msupply-foundation/open-msupply#3111]
Similar to the way we do sync with mSupply central (keep backwards compatible)
Would include adding to compatibility matrix on docs page

**Comment by nisha-dangol** [comment]
For build-testing, this basically means v2.1.0 central-oms syncs with remote oms-servers v2.0.0, v1.7.0, v1.6.0..., right? @andreievg @lache-melvin

**Comment by lache-melvin** [comment]
Hi @nisha-dangol  - yes it is for syncing between central OMS and remote OMS, but only from V2.0 onwards (per [docs](https://docs-staging.msupply.foundation/docs/introduction/requirements/#central-servers) OMS central not required until 2.0). 

Testing should confirm that Remote 2.0 and 2.1 can sync to Central 2.1 👍

**Comment by nisha-dangol** [comment]
Tested in v2.1.0-IA-test2 exe
Both remote 2.0.3 & remote 2.1.0 is syncing fine with central-oms 2.1.0.

I did notice that while initialising remote v2.0.3, sync almost stopped at `Pull Central` at some point (it literally stopped pulling records; for whole 1 hour, the record did not increase at all), so I restarted the app, then it picked up just fine. 

Could not replicate it on second time though. So will move to Done for now.
